README.md

```diff
@@ -188,6 +188,48 @@ db.Query("todos").UpdateById(docId, map[string]interface{}{"completed": true})
 db.Query("todos").DeleteById(docId)
 ```
 
+## Data Types
+
+Internally, CloverDB supports the following primitive data types: **int64**, **uint64**, **float64** and **time.Time**. When possible, values having different types are silently converted to one of the internal types: signed integer values get converted to int64, while unsigned ones to uint64. Float32 values are extended to float64.
```

Comment:
you could also add string to the primitive data types

Reply:
Yes, I forgot to add string and bool to the README